wazo_webhookd/plugins/subscription/http.py

```diff
@@ -136,6 +137,7 @@ def get(self, subscription_uuid):
         # NOTE:(sileht): To return 404 if the subscription doesn't exists
         self._service.get(subscription_uuid, self.visible_tenants())
 
-        results = list(self._service.get_logs(subscription_uuid))
+        filter_parameters = SubscriptionLogRequestSchema().load(request.args).data
+        results = list(self._service.get_logs(subscription_uuid, **filter_parameters))
```

Comment:
Why you cast to list? marshmallow can handle iterator

Reply:
When marshmallow consumes the iterator, the next `len()` won't work.

Reply:
Oh yeah sorry I was wrong. `query.all()` do not return an iterator but a list

Reply:
Why you won't fix?

Reply:
Why you won't fix?

Reply:
Oh right, I thought `query.all()` was an iterator, too.